lib/src/main/java/dataframe/Row.java

```diff
@@ -161,6 +164,18 @@ public boolean equals(Object o) {
         });
     }
 
+    public int compareOnColumn(Row row, String column){
+        var columnType = this.schema.get(column).dataType().getSimpleName();
+        return switch(columnType){
+            case "string": yield this.getString(column).compareTo(row.getString(column));
+            case "integer": yield Integer.compare(Integer.valueOf(this.getString(column)), Integer.valueOf(row.getString(column)));
+            case "double": yield Double.compare(Double.valueOf(this.getString(column)), Double.valueOf(row.getString(column)));
+            case "boolean": yield Boolean.compare(Boolean.valueOf(this.getString(column)), Boolean.valueOf(row.getString(column)));
+            case "float": yield Float.compare(Float.valueOf(this.getString(column)), Float.valueOf(row.getString(column)));
```

Comment:
We can create and use `<T>getAs` generic method for column name. 
Use the one created for column index as reference.

Reply:
That may throw error right now, need to do inferSchema strictly.